documentation/source/overview/testimonials.rst

```diff
@@ -0,0 +1,356 @@
+.. _testimonials:
+
+Testimonials
+############
+
+..
+    To get those testimonials, use $SCT_PATH/util/testimonials2rst
+
+Do you also use SCT and would like to share your feedback with the community? Please add your testimonial
+by clicking `here <https://forms.gle/Cfos8bRJTtRNxeMt8>`_. Thank you 😊
+
+
+.. admonition:: Christian Büchel, MD | Universitätsklinikum Hamburg-Eppendorf | Hamburg 🇩🇪
+
+   SCT is an invaluable tool for our investigations of central pain processing
+   in the spinal cord using fMRI. In addition, our combined cortico-spinal
+   imaging approach greatly benefits from the accurate spatial prepossessing
+   capabilities of the SCT.
+
+
+.. admonition:: Douglas Arnold, MD | NeuroRx & McGill University | Montreal 🇨🇦
+
+    Easily implemented. Beautiful output of state-of-the-art results
+
+
+.. admonition:: Claudia Gandini Wheeler-Kingshott, PhD | NMR Research Unit, UCL | London 🇬🇧
+
+   We use SCT for all our spinal cord analysis in studies of Multiple Sclerosis
+   as well as Spinal Cord Injury and ataxia. I also used SCT in the first study
+   of SC involvement in dementia.
```

Comment:
> @jcohenadad Did you have any plans to curate or sort the testimonials, or to make the more impactful ones stand out?

yes, i've already curated them by reordering and adding the city (and changed some affiliations). Do you think this is too much?  

also: do you mind going through the English and correcting "obvious" mistakes? Is that an acceptable thing to do? (i would like it if someone would fix a mistake that i unintentionally did when posting something online)

Reply:
> Is that an acceptable thing to do?

Hard to say... it depends on the person. But, I think your preference is likely to be similar to others. :) 

I get what you mean by "obvious" mistakes -- I won't go overboard, only small changes. :sweat_smile:

Reply:
> Do you think this is too much?

I think it might depend on the audience... For example, you personally recognize many of these people (and can tell which ones are "big names" so to speak), so it might be easier for certain testimonials to stand out.

But for me, I don't recognize many names, so they all have similar weight in my mind. (I can only really judge them by the content, so my preference is to remove some of the shorter/more generic testimonials and focus on the in-depth ones.)

Will the people reviewing funding proposals be able to recognize a testimonial's importance better than I can?

Reply:
> Will the people reviewing funding proposals be able to recognize a testimonial's importance based on a person's name?

yes, definitely. And we are quite lucky that about half of the testimonials are "big names".  

another thing is that i would feel bad for the "less big names" to have spent time and generosity filling the form and not having their name listed in the page (i think some would feel proud being listed there)

Reply:
> another thing is that i would feel bad for the "less big names" to have spent time and generosity filling the form and not having their name listed in the page (i think some would feel proud being listed there)

Very good point... I was speaking perhaps too much in the self-interest of SCT, when you're right, there is another side to the responses. :heart: